### PR TITLE
Feature/68 i os show the list of possible routes to take

### PR DESCRIPTION
**Pull request by jimibilibob**
* Draw polyline by color.
* Draw walking polyline.
* Fit possible route polyline on the map.
* Calculating a estimated time per possible route.
* Using Google Directions API.
* Method to sort the possible routes.
* King fisher added.
* Image helper implemented to cache images.
* Unit Test.

### CODE REVIEW

File: RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+//  Created by user on 19/9/22.
+//
+
+import Foundation
+import GoogleMaps
+
+class GoogleMapsHelper {
+    static var shared = GoogleMapsHelper()
+
+    func drawPolyline(map: GMSMapView, list: [Coordinate], hexColor: String = "#004696") {
```
Comment (jmacboy) [inline RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift]: add this default color as a constant

File: RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        let path = GMSMutablePath()
+        list.forEach({ path.add($0.toCLLocationCoordinate2D()) })
+        let polyline = GMSPolyline(path: path)
+        polyline.strokeColor = ColorHelper.shared.hexStringToUIColor(hex: hexColor)
+        polyline.strokeWidth = 5.0
+        polyline.geodesic = true
+        polyline.map = map
+    }
+
+    func drawDotPolyline(map: GMSMapView, path: GMSPath, hexColor: String = "#004696") {
```
Comment (jmacboy) [inline RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift]: add this default color as a constant

**Review by jmacboy — CHANGES_REQUESTED** [review]

File: RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift (hunk 1 above)
Comment (jimibilibob) [reply RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift]: Added the default polyline color as constant

File: RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift (hunk 2 above)
Comment (jimibilibob) [reply RoutesApp_ios/utilities/helpers/GoogleMapsHelper.swift]: Added the default polyline color as constant

**Review by jmacboy — APPROVED** [review]

**Pull request merged**